nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -8,7 +8,21 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 <!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
 
-- Create the first release note entry in this section!
+- A new option `systemd.sysusers.enable` was added. If enabled, users and
+  groups are created with systemd-sysusers instead of with a custom perl script.
+
+- A new option `system.etc.overlay.enable` was added. If enabled, `/etc` is
+  mounted via an overlayfs instead of being created by a custom perl script.
+
+- It is now possible to have a completely perlless system (i.e. a system
+  without perl). Previously, the NixOS activation depended on two perl scripts
+  which can now replaced via an opt-in mechanism. To make your system perlless,
```

Comment:
Some mistakes in the wording here.

Reply:
Can you point them precisely?

Reply:
> Can you point them precisely?

``which can now replaced`` doesn't make grammatical sense. I'm pretty sure right? :3

Reply:
missing them word "be"